jwst/tweakreg/tweakreg_step.py

```diff
@@ -70,18 +71,20 @@ def process(self, input):
             # entries when aligning to GAIA
             self.expand_refcat = True
 
+        if len(images) == 0:
+            raise ValueError("Input must contain at least one image model.")
```

Comment:
Is this necessary? I thought the code could already handle this albeit after some extra steps. I may be wrong.

Reply:
Good question!  I just realized that this check was done after generating the catalogs, so I just relocated from after to before catalogs are generated.  I suspect it is not needed.  Shall I remove?

Reply:
Ah, I see. No, don't change it: this is better.

Reply:
This check probably used to be at the top when this step was doing only alignment and we used to have a separate step to create catalogs. When the steps were merged - it was done sequentially and hence this check ended up somewhere away from the top.